en/syntax.tex

```diff
@@ -27,11 +27,12 @@ \subsection{Transitivity} Na'vi marks the subject of transitive and
 is gibberish.}\index{transitivity}
 
 \subsubsection{} Many compound verbs are created by pairing an
-uninflected noun or adjective with the verb stem \N{si} \E{to do, to
-make}, which is only used in these compounds, \N{irayo si} \E{to
-thank}, \N{kavuk si} \E{to betray}.  These verbs are always
-intransitive and use the dative for any object
-(\horenref{syn:case:dative-si}).
+uninflected noun or adjective, or occasionally an interjection,
+with the verb stem \N{si} \E{to do, to make}, which is only used 
+in these compounds, \N{irayo si} \E{to thank}, \N{kavuk si} \E{to betray}.
+Proper names may also be used in very casual conversations as slang.
```

Comment:
It may make more sense to put this in section 7.3.3 on the colloquial register?

Reply:
Yeah, that makes sense. Somehow I forgot that there was a section on that

Reply:
You're not alone hrh